src/views/Challenge.vue

```diff
@@ -79,17 +80,33 @@ export default {
         rating = 'rating'
         no_ass = 'no_assessments'
       }
-      return [
+      
+      const default_obj = [
         { k: { v: 'amount', r: false}, l: this.$t('challenge.REQUESTED_BUDGET_DESC')},
         { k: { v: 'amount', r: true}, l: this.$t('challenge.REQUESTED_BUDGET_ASC')},
         { k: { v: rating, r: false}, l: this.$t('challenge.REVIEW_SCORE_DESC')},
         { k: { v: rating, r: true}, l: this.$t('challenge.REVIEW_SCORE_ASC') },
         { k: { v: 'title', r: false}, l: this.$t('challenge.AZ') },
         { k: { v: 'title', r: true}, l: this.$t('challenge.ZA') },
         { k: { v: no_ass, r: false}, l: this.$t('challenge.NO_REVIEWS_DESC') },
-        { k: { v: no_ass, r: true}, l: this.$t('challenge.NO_REVIEWS_ASC') },
-        { k: { v: 'random', r: false}, l: this.$t('challenge.RANDOM') },
+        { k: { v: no_ass, r: true}, l: this.$t('challenge.NO_REVIEWS_ASC') }
       ]
+      let exists_vcast = proposals.filter((el) => el.votes_cast);
```

Comment:
The solution is correct and I like the fact that the existence of this additional fields is based on the data structure itself, it will be useful for future updates and it's always good to non hardcode conditions.
The only thing to consider is that filter in this way will loop every proposal in the challenge (and because it's in a computed property it can happen dinamically).
For this reason, assuming that the data structure is the same for each element, I would create a new computed properties for these conditions checking only the first element of `this.proposals`. Something like:
```
withVoteCast(){
  if (this.proposals.length) {
    return this.proposals[0].votes_cast
  }
  return false
}
```

It will increase the performance maintaining the same logic.